src/lib.rs

```diff
@@ -564,6 +600,33 @@ impl <'a> BitOrAssign for FixedBitSet
     }
 }
 
+impl <'a> BitXor for &'a FixedBitSet
+{
+    type Output = FixedBitSet;
+    fn bitxor(self, other: &FixedBitSet) -> FixedBitSet {
+        let (short, long) = {
+            if self.len() <= other.len() {
+                (&self.data, &other.data)
+            } else {
+                (&other.data, &self.data)
+            }
+        };
+        let mut data = long.clone();
+        for (i, &block) in short.iter().enumerate() {
```

Comment:
Can you use `zip` here? It is more idiomatic, safer, and potentially more performant.

Reply:
Good idea. The latest commit changes BitAnd, BitOr, and BitXor to use `zip` here.